test/integration/testManualConnection.js

```diff
@@ -70,6 +72,40 @@ if (process.env.RUN_MANUAL_TESTS_ONLY === 'true') {
           }
         });
       });
+
+      describe('Connection - ID Token authenticator', async function (done) {
+        const key = Util.buildCredentialCacheKey(connectionOption.host, connectionOption.username, 'ID_TOKEN');
+        GlobalConfig.getCredentialManager().remove(key);
+        
+        const connectionOption = connOption.externalBrowser;
+
+        it('test - obtain the id token from the server and save it on the local storage', async function () {
+          const connection = snowflake.createConnection(connectionOption);
+          await connection.connectAsync(function (err) {
+            assert.ok(!err);
+            const idToken = GlobalConfig.getCredentialManager().read(key);
```

Comment:
should we call the functions from `GlobalConfig.getCredentialManager()` with await?

Reply:
I fixed it